### PR TITLE
add second donate button to campaign view

### LINKED ISSUE #1405: Add a second Donate button at the end of the description of a Campaign
Many people might start reading what they're donating to, at the bottom, before Milestones appear there should be a second button saying: 'Donate'

image

### PR DESCRIPTION
**Pull request by mahdine**
fixes #1405

### CODE REVIEW

File: src/components/views/ViewCampaign.jsx [hunk 1]
```diff
@@ -339,7 +339,20 @@ class ViewCampaign extends Component {
                       <div className="card content-card ">
                         <div className="card-body content">{this.renderDescription()}</div>
                       </div>
-
+                      <div className="text-center">
+                        <div style={{ transform: 'translateY(-50%)' }}>
+                          <DonateButton
```
Comment (aminlatifi) [inline src/components/views/ViewCampaign.jsx:344]: @mahdine Add the disable auto popup property to this DonateButton component, we need just one Donate Button with auto popup one page

**Comment by aminlatifi** [comment]
@mahdine find my comment on source code.

**Comment by GriffGreen** [comment]
Screen Shot 2020-08-27 at 6 27 02 PM
Button placement looks off

**Comment by aminlatifi** [comment]
LGTM

**Pull request merged**